src/Translations.js

```diff
@@ -25,13 +32,25 @@ const ENGLISH_TRANSLATION = {
         "en": "English",
         "nl": "Nederlands"
     },
+    "input_modes": {
+        "switch_to_interactive": "Switch to interactive mode",
+        "switch_to_batch": "Switch to batch input"
+    },
+    "enter": "Enter"
 };
 
 const DUTCH_TRANSLATION = {
     "Papyros": "Papyros",
-    "enter_code": "Voer hier je code in",
-    "enter_input": "Geef hier de invoer voor je code in",
-    "code_output": "Resultaat van de code",
+    "code": "Code",
+    "code_placeholder": "Voer hier je code in",
+    "input": "Invoer",
+    "input_placeholder": {
+        "interactive": "Geef hier je invoer in. Duw op enter om te verzenden.",
```

Comment:
What about using the actual prompt from `input('Question: ')` here (with a default text as fallback if no prompt (empty string) is given, instead of putting the prompt in the output panel? This way, the right question is asked, and the output panel is not cluttered with the prompts.